### PR TITLE
Is31/stabilise platform

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- [x] updated aio-pika to 6.5.2
- [x] removed socket.io asynciopikamanager that was creating a lot of connections/channels/queues without removing them
- [x] ensure consumed queue on webserver side does not need to be acknowledged (should prevent memory from raising in rabbit to keep messages)
- [x] set sidecar number to 4 for development mode
- [x] some cleanup in the sidecar code to ensure pika connections are correctly closed

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: services/docker-compose.yml [hunk 1]
```diff
@@ -82,9 +82,10 @@ services:
 
   sidecar:
     image: ${DOCKER_REGISTRY:-itisfoundation}/sidecar:${DOCKER_IMAGE_TAG:-latest}
-    init: true
+    init: true
```
Comment (odeimaiz) [inline services/docker-compose.yml]:
```suggestion
    init: true
```

File: services/web/server/src/simcore_service_webserver/config/server-defaults.yaml [hunk 2]
```diff
@@ -15,10 +15,7 @@ director:
   port: 8001
   version: v0
 socketio:
-  enabled: True
-  message_queue:
-    user: simcore
-    password: simcore
+  enabled: True
```
Comment (odeimaiz) [inline services/web/server/src/simcore_service_webserver/config/server-defaults.yaml]:
```suggestion
  enabled: True
```

File: services/web/server/src/simcore_service_webserver/socketio/__init__.py [hunk 3]
```diff
@@ -19,11 +19,6 @@
 def setup(app: web.Application):
     mgr = None
     cfg = app[APP_CONFIG_KEY][CONFIG_SECTION_NAME]
-    if "message_queue" in cfg and cfg["message_queue"]:
-        mq_config = cfg["message_queue"]
-        url = f"amqp://{mq_config['user']}:{mq_config['password']}@{mq_config['host']}:{mq_config['port']}"
-        mgr = AsyncAioPikaManager(url=url, logger=log)
-
     sio = AsyncServer(async_mode="aiohttp", client_manager=mgr, logging=log)
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/socketio/__init__.py]: does this server shuts down correctly as is?

Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/socketio/__init__.py]: that is a good question...

**Pull request merged**